### PR TITLE
[4.x] add PHP 8.3 to tests

### PR DESCRIPTION
**Pull request by heelc29**
### Summary of Changes

[PHP 8.3 is released](https://www.php.net/releases/8.3/en.php)

### Testing Instructions

Drone CI/CD

### Actual result BEFORE applying this Pull Request

PHP 8.3 is not tested

### Expected result AFTER applying this Pull Request

PHP 8.3 should be tested

### Additional comments

Drone needs to be signed

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -6,15 +6,15 @@ clone:
 
 steps:
   - name: setup
-    image: joomlaprojects/docker-images:php8.2
+    image: joomlaprojects/docker-images:php8.3
```
Comment (richard67) [inline .drone.yml:9]: Do we really want to move to 8.3 also here? The recommended version is still 8.2, minimum is 8.1.

File: .drone.yml [hunk 2]
```diff
@@ -6,15 +6,15 @@ clone:
...
     volumes:
       - name: certificates
         path: /certificates
     commands:
       - cp -v tests/certs/* /certificates/
 
   - name: composer
-    image: joomlaprojects/docker-images:php8.2
+    image: joomlaprojects/docker-images:php8.3
```
Comment (richard67) [inline .drone.yml:17]: Do we really want to move to 8.3 also here? The recommended version is still 8.2, minimum is 8.1.

File: .drone.yml [hunk 3]
```diff
@@ -230,7 +251,7 @@ steps:
     depends_on:
       - phpmin-system-mysql
       - phpmin-system-postgres
-    image: joomlaprojects/docker-images:cypress8.1
+    image: joomlaprojects/docker-images:cypress8.3
```
Comment (richard67) [inline .drone.yml:254]: Do we really want to move to 8.3 also here? Or should we stay with the minimum 8.1?

File: .drone.yml [hunk 4]
```diff
@@ -243,7 +264,7 @@ steps:
...
```
Comment (richard67) [inline .drone.yml:267]: Do we really want to move to 8.3 also here? Or should we stay with the minimum 8.1?

File: .drone.yml [hunk 5]
```diff
@@ -269,7 +290,7 @@ steps:
     depends_on:
       - phpmax-system-mysql
       - phpmax-system-postgres
-    image: joomlaprojects/docker-images:cypress8.1
+    image: joomlaprojects/docker-images:cypress8.3
```
Comment (richard67) [inline .drone.yml:293]: Do we really want to move to 8.3 also here? Or should we stay with the minimum 8.1?

File: .drone.yml (hunk 1 above)
Comment (laoneo) [reply .drone.yml:9]: I don't have an issue going with the latest when it runs fine.

Comment (richard67) [reply .drone.yml:9]: Well, drone is passing so it seems to run fine. So if you want, mark my comments as resolved.

**Comment by richard67** [comment]
> @heelc29 can you remove the draft status so I can merge and will afterwards update the drone key.

@laoneo We can do that, too. At least for me the button "Ready for review" is enabled.

**Comment by heelc29** [comment]
> @heelc29 can you remove the draft status so I can merge and will afterwards update the drone key.

Done

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**